examples/QIR/Development/Development.csproj

```diff
@@ -9,8 +9,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20220129-005156" PrivateAssets="All" GeneratePathProperty="true"/>
-    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.24.217480-alpha" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.24.217480-beta" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Quantum.Qir.Runtime" Version="0.25.222597-alpha" GeneratePathProperty="true" />
```

Comment:
Should this be non-alpha?

Reply:
There is a special case for 'Microsoft.Quantum.Qir.Runtime' that the package is published as -alpha, so we need to treat it differently. See the version in NuGet:

https://www.nuget.org/packages/Microsoft.Quantum.Qir.Runtime